oxide_core/Cargo.toml

```diff
@@ -10,7 +10,7 @@ license = "MIT"
 edition = "2018"
 
 [lib]
-name = "oxide_render"
+name = "oxide_core"
```

Comment:
I think we should fix this.

Reply:
Isn't it already fixed?

Reply:
It meant moving from oxide_core to oxide_math.